src/components/Errors/Error.tsx

```diff
@@ -0,0 +1,32 @@
+import classNames from 'classnames';
+
+type Props = {
+  error: boolean,
+  setError: (param: boolean) => void,
+  errorMessage: string,
+};
+
+export const Error: React.FC<Props> = ({
+  error,
+  errorMessage,
+  setError,
+}) => {
```

Comment:
automatically hide the notification in 3 seconds;

you need to create setTimeout on the first render and clear timeout on unmount, use useEffect